projects/1.16.1/assets/tetra/tetra/lang/zh_cn.json

```
@@ -0,0 +1,2083 @@
+{
+    "itemGroup.tetra": "Tetra",
+    "tetra.attribute.draw_strength.name": "拉弓力量",
+    "tetra.attribute.draw_speed.name": "拉弓耗时",
+    "tetra.attribute.ability_damage.name": "特殊功能伤害",
+    "tetra.attribute.ability_cooldown.name": "特殊功能冷却时间",
+    "item.tetra.holo.tooltip1": "轻触一下，球体便将图和文字投射到空中",
+    "item.tetra.holo.tooltip2": "§8[§7右键§8] §7使用",
+    "tetra.holo.craft.breadcrumb.root": "合成",
+    "tetra.holo.craft.breadcrumb.materials": "材料",
+    "tetra.holo.craft.materials": "材料",
+    "tetra.holo.craft.modular_sword": "刀剑",
+    "tetra.holo.craft.modular_toolbelt": "工具腰带",
+    "tetra.holo.craft.modular_double": "双头工具",
+    "tetra.holo.craft.modular_single": "单头工具",
+    "tetra.holo.craft.modular_bow": "弓",
+    "tetra.holo.craft.modular_shield": "盾牌",
+    "tetra.holo.craft.slot": "插槽",
+    "tetra.holo.craft.requirements": "需要",
+    "tetra.holo.craft.improvements": "改进",
+    "tetra.holo.craft.stats": "属性",
+    "tetra.holo.craft.variants": "变种",
+    "tetra.holo.craft.materials.stat.primary": "硬度",
+    "tetra.holo.craft.materials.stat.primary.tooltip": "材料的硬度、锋利度，或被塑形成硬质/锋利边缘的能力",
+    "tetra.holo.craft.materials.stat.secondary": "密度",
+    "tetra.holo.craft.materials.stat.secondary.tooltip": "材料的密度、刚性或重量。被挥动时携带的动大小。",
```

|Feedback Priority|
|---|
|⛰ Mountain / **Blocking and requires immediate action**|

```suggestion
    "tetra.holo.craft.materials.stat.secondary.tooltip": "材料的密度、刚性或重量。被挥动时携带的动能大小",
```